### DISCUSSION TITLE
Can your team give the training.py in github?

### DISCUSSION THREAD

**Discussion by nakuda1** [obss/sahi#822] [Q&A] [answered]
Sahi is a great tool. If your team can give the training.py and related document in github, we can add different backbones and train our own model. Then our best .pt can be uesd to do more work with sahi.

**Comment by fcakyon** [comment, marked as answer]
You can train any model with yolov5 framework or mmdetection framework. Here is an example with mmdetection: https://github.com/fcakyon/small-object-detection-benchmark

**Reply by nakuda1** [reply]
Thanks, I solved it!
